src/uct/tcp/tcp.h

```diff
@@ -145,8 +145,8 @@ typedef struct uct_tcp_iface_config {
     int                           prefer_default;
     unsigned                      max_poll;
     int                           sockopt_nodelay;
-    int                           sockopt_sndbuf;
-    int                           sockopt_rcvbuf;
+    size_t                        sockopt_sndbuf;
```

Comment:
need to change also uct_tcp_iface::sockopt to size_t

Reply:
yes, thanks 👍